Server/mods/deathmatch/logic/packets/CEntityAddPacket.cpp

```diff
@@ -959,6 +959,25 @@ bool CEntityAddPacket::Write(NetBitStreamInterface& BitStream) const
                         BitStream.Write(currentWeaponSlot);
                     }
 
+                    // Animation
+                    if (BitStream.Can(eBitStreamVersion::AnimationsSync))
+                    {
+                        const SPlayerAnimData& animData = pPed->GetAnimationData();
+
+                        // Contains animation data?
+                        BitStream.WriteBit(!animData.blockName.empty() && !animData.animName.empty());
+
+                        BitStream.WriteString(animData.blockName);
```

Comment:
You forgot a condition here. You can see that client doen't expect the data if a bit is set to zero.